props/saber_BC_buttons.h

```diff
@@ -2057,13 +2071,16 @@ void DoSavedTwist() {
   }
 
   void NextBladeArray() {
+  #ifdef BLADE_ID_SCAN_MILLIS
     if (!use_fake_id_) best_config_before_faking_ = current_config - blades;
-    size_t next_array = (current_config - blades + 1) % NELEM(blades);
-    fake_id_ = blades[next_array].ohm;
+    fake_id_ = blades[((current_config - blades) + 1) % NELEM(blades)].ohm;
     use_fake_id_ = true;
-
     FindBladeAgain();
     PlayArraySound();
+  #else
+    current_config = blades + (((current_config - blades) + 1) % NELEM(blades));
```

Comment:
This won't work.
This doesn't tear down blades, change directory or do any of the things that FindBladeAgain() does.

Also, I'm not sure what BLADE_ID_SCAN_MILLIS has to do with it... It should work anyways, no?

Reply:
ugh I left out FindBladeAgain(). I meant to have both calls outside the #endif.
However, that's not all the problem.

This PR is a result of me not testing thoroughly enough.
The way it stands, if BLADE_ID_SCAN_MILLIS is not defined, BCprop tries to override GetNoBladeLevelBefore(), but that doesn't exist.
If just adding a guard to my override to try to fix that, NextBladeArray() breaks.
 
Also, I hadn't tested using multiple preset arrays and different fonts, so it "worked" as I first submitted here, but then found the issues.
 
I've since refactored it, tested "all" the scenarios, and it seems correct now.
Does it look right / make sense?